components/authz-service/server/v2/policy_test.go

```diff
@@ -2981,11 +2980,9 @@ func setupV2WithMigrationState(t *testing.T,
 	polV2, _, err := v2.NewPoliciesServer(ctx, l, mem_v2, writer, pl, vChan)
 	require.NoError(t, err)
 
-	eventServiceClient := &testhelpers.MockEventServiceClient{}
-	configMgr, err := config.NewManager("/tmp/.authz-delete-me")
 	require.NoError(t, err)
 	projectsSrv, err := v2.NewProjectsServer(ctx, l, mem_v2,
-		rulesRetriever, eventServiceClient, configMgr, testhelpers.NewMockPolicyRefresher())
+		rulesRetriever, testhelpers.NewMockProjectUpdateManager(), testhelpers.NewMockPolicyRefresher())
```

Comment:
I wished to mock the project update manager itself, greatly simplifying the necessary test framework by allowing focussing just on the DB. That necessitated this change in a few places. 

All the changes herein other than rules_property_test.go are part of this adjustment.